### PR TITLE
Slovak translation

### PR DESCRIPTION
**Pull request by agiat86**

### CODE REVIEW

**Comment by ihhub** [comment]
Hi @agiat86 , please fix merge conflict here.

File: files/lang/sk.po [hunk 1]
```diff
@@ -9699,38 +9689,38 @@ msgid ""
 msgstr ""
 
 msgid "Meteor Shower"
-msgstr ""
+msgstr "Meteorický roj"
```
Comment (Roxterat) [inline files/lang/sk.po:9692]: "Roj / (s)pád meteorov" could fit here perhaps.

File: files/lang/sk.po [hunk 2]
```diff
@@ -9699,38 +9689,38 @@ msgid ""
...
+msgstr "Prstenec chladu"
 
 msgid ""
 "Drains body heat from all units surrounding the center point, but not "
 "including the center point."
 msgstr ""
 
 msgid "Disrupting Ray"
-msgstr ""
+msgstr "Narušujúci lúč"
```
Comment (Roxterat) [inline files/lang/sk.po:9723]: "Rušivý lúč" maybe..

**Review by Roxterat — CHANGES_REQUESTED** [review]
@agiat86 left some comments, mostly alternatives.

File: files/lang/sk.po [hunk 3]
```diff
@@ -9440,7 +9430,7 @@ msgid ""
 msgstr ""
 
 msgid "Sword Breaker"
-msgstr ""
+msgstr "Lamač mečov"
```
[the remark this answers is not in the document]
Comment (agiat86) [reply files/lang/sk.po:9433]: Oh, yeah, thanks.

File: files/lang/sk.po (hunk 2 above)
Comment (agiat86) [reply files/lang/sk.po:9723]: Sounds too much as "disturbing ray" and "cancelling ray" so idk

File: files/lang/sk.po [hunk 4]
```diff
@@ -9609,61 +9599,61 @@ msgid "Causes all of your units to inflict maximum damage."
...
 "Damages all undead in the battle.  This is an improved version of Holy Word."
 msgstr ""
 
 msgid "Holy Shout"
-msgstr ""
+msgstr "Svätý výkrik"
 
 msgid "Anti-Magic"
-msgstr ""
+msgstr "Anti-Mágia"
```
[the remark this answers is not in the document]
Comment (agiat86) [reply files/lang/sk.po:9644]: It could. Yeah, it sound kind of cool too!

File: files/lang/sk.po (hunk 1 above)
Comment (agiat86) [reply files/lang/sk.po:9692]: Pád meteorov? Meteor Shower meaning Meteorický roj is a correct translation but it's not really what's happening. It's really a small meteor disaster, instead. The russian translation is Камнепад. We can go with that. Pád kameňov. They are rocks, after all. Nice and short too. We can revisit that if needed.

**Review by Roxterat — APPROVED** [review]
Approved.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@agiat86 , thank you very much for this update!

**Pull request merged**